### DISCUSSION TITLE
How to connect my local kafka to kafka-ui running in docker.

### DISCUSSION THREAD

**Discussion by psquared-dev** [provectus/kafka-ui#4450] [Q&A]
As the title, says I want to connect my local kafka installation (NOT INSIDE DOCKER), to kafka-ui running in docker container?

**Comment by waitspring** [comment]
In your kafka-ui docker's yaml file, you can find some parameters like:
```
# kafka:
#   clusters:
#     -
#       name: local
#       bootstrapServers: localhost:9092
#       metrics:
#         port: 9094
#         type: JMX
```
Taking `localhost` to your kafka host IPv4 address, and kafka-ui docker can connect kafka server.

**Comment by germanosin** [comment]
Me and @Haarolean  no longer support this repo, don't you mind to post this to https://github.com/kafbat/kafka-ui
